message_definitions/v1.0/development.xml

```diff
@@ -38,5 +47,29 @@
       <field type="float" name="raw_press" units="hPa">Raw differential pressure. NaN for value unknown/not supplied.</field>
       <field type="uint8_t" name="flags" enum="AIRSPEED_SENSOR_FLAGS">Airspeed sensor flags.</field>
     </message>
+    <message id="420" name="RADIO_RC_CHANNELS">
+      <description>RC channel outputs from a MAVLink RC receiver for input to a flight controller or other components (allows an RC receiver to connect via MAVLink instead of some other protocol such as PPM-Sum or S.BUS).
+        Note that this is not intended to be an over-the-air format, and does not replace RC_CHANNELS and similar messages reported by the flight controller.
+        The target_system field should normally be set to the system id of the system to control, typically the flight controller.
+        The target_component field can normally be set to 0, so that all components of the system can receive the message.
+        The channels array field can publish up to 32 channels; the number of channel items used in the array is specified in the count field.
+        The time_last_update_ms field contains the timestamp of the last received valid channels data in the receiver's time domain.
+        The count field indicates the first index of the channel array that holds invalid data.
```

Comment:
It's invalid data from the perspective MAVLink - which uses the term to mean "this isn't usable data".
But since you don't like this:

```suggestion
        The count field indicates the first index of the channel array that is not used for channel data (this an later indexes are zero-filled).
```

Reply:
@hamishwillee Text is good but small typo I think?  Should be '(this and later indexes...' ?

Reply:
Whoop. Yes, fixed.